featurebyte/api/feature.py

```diff
@@ -374,18 +376,22 @@ def _list_versions(
         --------
         List saved Feature versions
 
-        >>> Feature.list_versions()  # doctest: +SKIP
-            name        version  dtype readiness  online_enabled             table    entities              created_at
-        0  new_feat2  V230323  FLOAT     DRAFT           False      [sf_event_table]  [customer] 2023-03-23 07:16:21.244
-        1  new_feat1  V230323  FLOAT     DRAFT           False      [sf_event_table]  [customer] 2023-03-23 07:16:21.166
-        2     sum_1d  V230323  FLOAT     DRAFT           False      [sf_event_table]  [customer] 2023-03-23 07:16:21.009
+        >>> Feature.list_versions(include_id=False)  # doctest: +ELLIPSIS
+                                     name  version      dtype         readiness  online_enabled                                          tables    primary_tables           entities   primary_entities  created_at  is_default
+        0  CustomerLatestInvoiceTimestamp      ...  TIMESTAMP  PRODUCTION_READY           False                                [GROCERYINVOICE]  [GROCERYINVOICE]  [grocerycustomer]  [grocerycustomer]         ...  True
```

Comment:
Would it be better if we can check the full output? I think we can customise the clipping behaviour by setting the right pandas options maybe in conftest:

```
import pandas as pd
pd.set_option('display.max_rows', 500)
pd.set_option('display.max_columns', 500)
pd.set_option('display.width', 1000)
```

Reply:
This the full output. The main reason why `...` is used for the version is that it is sensitive to the date we run the test (version name contains date info). Including the version name will cause the test to pass only today but fail after that.